collectors/hpa.go

```diff
@@ -21,7 +21,7 @@ import (
 
 	"github.com/golang/glog"
 	"github.com/prometheus/client_golang/prometheus"
-	autoscaling "k8s.io/api/autoscaling/v1"
+	autoscaling "k8s.io/api/autoscaling/v2beta1"
```

Comment:
@discordianfish Why do we change api version for autoscaling from `v1` to `v2beta1`?

Reply:
Is `v2beta1` the newest api version for autoscaling?

Reply:
The Conditions field wasn't available in v1 and according to @DirectXMan12 and @brancz's it's okay to update it since it's been supported for a while (iirc 1.6?)

Reply:
So `v2beta1` is the newest api version for autoscaling?

Reply:
I think so, yes: https://kubernetes.io/docs/tasks/run-application/horizontal-pod-autoscale/#api-object